ansible/roles_ocp_workloads/ocp4_workload_quarkus_workshop_user/defaults/main.yml

```diff
@@ -4,4 +4,15 @@ ocp_username: jfalkner-redhat.com
 silent: False
 
 ocp4_workshop_quarkus_workshop_user_che_user_password: 'openshift'
+ocp4_workshop_quarkus_workshop_user_openshift_user_password: 'openshift'
```

Comment:
i usually suggest to set password to something generated by default, and then override it in agnosticv later

```yaml
ocp4_workshop_quarkus_workshop_user_openshift_user_password:: "{{ lookup('password', '/dev/null length=12 chars=ascii_letters,digits') }}"
```

Reply:
good idea. will fix.

Reply:
Fixed! I've overridden the value in AgnosticV here.